cheatsheets/Security_Definitions_Cheat_Sheet.md

```diff
@@ -0,0 +1,29 @@
+# Security Definitions Cheat Sheet
+
+## Encoding vs Escaping
+
+**Baseline Definition:**  
+- **Encoding**: Transforming data into another format using a publicly available scheme (e.g., Base64, URL encoding).  
+- **Escaping**: Adding special characters to data so that it is treated as plain text, not as executable code (e.g., HTML entity escaping).  
+- Sources: [OWASP](https://owasp.org), [CNCF Glossary](https://glossary.cncf.io)
+
+**Why it Matters in Security:**  
+- Misunderstanding these terms can cause developers to use the wrong defense mechanism.  
+- Encoding does **not** protect against injection attacks, while escaping is often critical for preventing XSS.  
```

Comment:
While generally true, this is likely to send a lot of developers into a panic because we call most of these 'Encoders' nor 'Escapers'. E.g., https://javadoc.io/static/org.owasp.encoder/encoder/1.3.1/org/owasp/encoder/Encode.html and https://javadoc.io/static/org.owasp.esapi/esapi/2.7.0.0/org/owasp/esapi/Encoder.html. And while one could argue that these both perhaps misnamed, neither of them are going to get renamed at this point.